pkg/virt-handler/BUILD.bazel

```diff
@@ -152,6 +152,7 @@ go_test(
 
 ginkgo_test(
     name = "go_parallel_test",
+    timeout = "long",
```

Comment:
Can you provide 1/2 links that shows this pkg is also timeout-ing?

Reply:
I'm likely missing something here but where is this used by the rule?

https://github.com/kubevirt/kubevirt/blob/611d4ed08b797cfe937834196e1a0fa01ee2cb7c/tools/ginkgo/ginkgo.bzl#L33-L39